src/send_data.py

```diff
@@ -0,0 +1,51 @@
+from ast import alias
+from venv import create
+from xmlrpc.client import boolean
+from sensor_temp import getTemp
+from sensor_motion import getMotion 
+from alarm import alarm_on
+from button import button_pressed
+import requests
+import json
+import time
+import datetime
+
+URL = 'http://127.0.0.1:8000'
+DEVICE_ID = '1'
+
+## SEND DATA OUT   
+def create_sensor_data_dict():
+    sensor_temp = getTemp()
+
+    try:
+        temp_c = round(sensor_temp["temperature_c"], 1)
+        temp_f = round(sensor_temp["temperature_f"], 1)
+        humidity = sensor_temp["humidity"]
+        motion = getMotion()
+        alarm = alarm_on()
+        button = button_pressed()
+        timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+
+        sensor_data_out = {
+            'temperature_c': temp_c,
+            'temperature_f': temp_f,
+            'humidity': humidity,
+            'motion': motion,
+            'alarm': alarm,
+            'button': button,
+            'created_at': timestamp
+        }
+
+        return sensor_data_out
```

Comment:
You can immediately return this dictionary.
```python
return {
    'temperature_c': temp_c,
    'temperature_f': temp_f,
    'humidity': humidity,
    'motion': motion,
    'alarm': alarm,
    'button': button,
    'created_at': timestamp
}
```

Reply:
Done